core/importer/datasets_importer.py

```diff
@@ -496,19 +496,31 @@ def _process_study(study):
             description = study.get('description', '')
             has_ethics_approval = study.get('has_ethics_approval', False)
             ethics_approval_notes = study.get('ethics_approval_notes', '')
-            url = study.get('url', '')  # TODO: Currently this is lost
+            url = study.get('url', '')
 
-            cohort, _ = Cohort.objects.get_or_create(
```

Comment:
Why not get_or_create by just title? The more you increase parameters to get_or_create the more likely you will create new records for same cohort.